driver/csiplugin/controllerserver.go

```diff
@@ -1176,7 +1189,20 @@ func (cs *ScaleControllerServer) CreateVolume(newctx context.Context, req *csi.C
 		if capRange == nil {
 			return nil, status.Error(codes.InvalidArgument, "volume range is not provided")
 		}
-		capacity := uint64(capRange.GetRequiredBytes()) // #nosec G115 -- false positive
+		// #nosec G115 -- false positive
+		// changing capacity here for pvc size in decimal units to align with scale block size
+		capacity := uint64(capRange.GetRequiredBytes())
+		filesystemname := scaleVol.VolBackendFs
```

Comment:
better to follow camelCase/pascalCase for the variable names.
```suggestion
		filesystemName := scaleVol.VolBackendFs
```

Reply:
@badri-pathak done above change for all the variables I declared